chapters/object_types/spatial_fields.txt

```diff
@@ -53,3 +53,38 @@ Structured regular fields only need to store the values of the samples, because
 their addresses in memory can be easily computed from a 3D position. A
 common type of structured spatial fields are regular grids.
 ====
+
+[[object_types_spatial_field_amr]]
+==== Adaptive Mesh Refinement 
+
+Extension `KHR_SPATIAL_FIELD_AMR`
+
+Adaptive Mesh Refinement (AMR) spatial fields are created by passing the subtype string
+`amr` to <<object_types_spatial_field, `anariNewSpatialField`>>. The parameters
+understood by AMR spatial fields are summarized in the table below.
+
+.Configuration parameters for AMR spatial fields.
+[cols="<,<2,>,<5",options="header,unbreakable"]
+|===================================================================================================
+| Name         | Type                     | Default   | Description
+| block.data   |`ARRAY1D` of `ARRAY3D` of `FLOAT32` | | array of 3D arrays containing the scalar voxel data 
+| gridOrigin   |`FLOAT32_VEC3`            | (0, 0, 0) | origin of the grid in object-space
+| gridSpacing  |`FLOAT32_VEC3`            | (1, 1, 1) | size of the grid cells in object-space
+| method       |`STRING`                  | `current` | sampling method, possible values are: `current`, `finest`, `octant` 
```

Comment:
After discussing with Stefan Zellmann, having the method be a future extension proposed by implementations that need it, like OSPRay, is probably the better choice here since it is more of a back-end specific parameter.